src/NearOrg/Ecosystem/GetFundingPage.jsx

```diff
@@ -318,10 +416,12 @@ return (
             {fundingCards.map((item) => (
               <div className="col" key={item.key}>
                 <Card center>
-                  <SocialIcon className={item.iconClassName} color={`var(--${item.iconColor})`} />
-                  <Text size="text-m" color="sand12">
-                    {item.content}
-                  </Text>
+                  <a href={`#${item.key}`}>
```

Comment:
I'd like to see `<Link>` instead of `<a>` and it seems we can use it -> [doc](https://nextjs.org/docs/pages/api-reference/components/link#disable-scrolling-to-the-top-of-the-page). On the other hand it's just a navigation on the page so this comment could be ignored

Reply:
@charleslavon this comment is optional so feel free to drop this